docs/user-guide/liveness/http-liveness.yaml

```diff
@@ -13,6 +13,8 @@ spec:
       httpGet:
         path: /healthz
         port: 8080
+        httpHeaders:
+          - ['X-Custom-Header', 'Awesome']
```

Comment:
The alternative format in that doc is for a list, but we need to
marshal/unmarshal as a struct (map in yaml).  It might work but it's
surprising at best.  Thanks :)

Ping me when all done and I'll final review ASAP.  This will make it into
1.2.

On Fri, Jan 29, 2016 at 9:54 AM, Rudi C <notifications@github.com> wrote:

> In docs/user-guide/liveness/http-liveness.yaml
> <https://github.com/kubernetes/kubernetes/pull/20116#discussion_r51291887>
> :
>
> > @@ -13,6 +13,8 @@ spec:
> >        httpGet:
> >          path: /healthz
> >          port: 8080
> > +        httpHeaders:
> > +          - ['X-Custom-Header', 'Awesome']
>
> It's at the bottom of my YAML bible, Complete idiot's introduction to YAML
> <https://github.com/Animosity/CraftIRC/wiki/Complete-idiot's-introduction-to-yaml>.
> Fixed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/20116/files#r51291887>.
>